deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -331,6 +331,13 @@ cephClusterSpec:
       requests:
         cpu: "500m"
         memory: "100Mi"
+    exporter:
```

Comment:
it is ceph exporter?

Reply:
@parth-gr Yes, https://github.com/rook/rook/blob/ae0823e52aa717ff816242ecd0b5ebd8c8405c83/pkg/operator/ceph/cluster/nodedaemon/exporter.go#L179